src/lib/application/application.factory.ts

```diff
@@ -62,8 +62,8 @@ function transform(options: ApplicationOptions): ApplicationOptions {
  * but only the rule *1* is addressed by this function as the other ones doesn't
  * have a canonical representation.
  */
-function resolvePackageName(path: string) {
-  const { base: baseFilename, dir: dirname } = parse(path);
+function resolvePackageName(path: string | number) {
```

Comment:
I believe we should keep this as `path: string` only

Reply:
it's just a suggestion, now you can generate a project called 360

Reply:
I got it but instead of changing the signature of `resolvePackageName`, you can call  
`target.name = resolvePackageName(target.name.toString());`